app/assets/stylesheets/layouts/_notifications.scss

```diff
@@ -0,0 +1,54 @@
+$notification-radius: 5px;
+
+.global-alert {
+  border-radius: $notification-radius;
+  margin-top: 1.5em;
```

Comment:
Think the preference is to use `$baseline-unit` (or a multiple of) instead of `em` values